site/topics/genetic-operators/genetic-operators.rst

```diff
@@ -7,6 +7,103 @@ Genetic Operators
 Genetic Operators for Binary Representations
 ============================================
 
+* Binary representations are those that only contain two values, which are typically ``0``\s and ``1``s
```

Comment:
some kind of typo here. one is `\s` the other is only `s`. a lot of mixing of these throughout. enforcing numbers to be formatted as code is probably overkill